packages/adminrouter/extra/src/includes/server/master.conf

```diff
@@ -23,6 +23,32 @@ location / {
     include /opt/mesosphere/etc/adminrouter-ui-security.conf;
 }
 
+location /assets {
+    # Offer gzip compression for most common MIME types. This is especially relevant
+    # for the DC/OS UI, which is served by Master Admin Router and consumed by
+    # browsers. However, on the client side this also applies to non-browser
+    # consumers within the cluster (such as native DC/OS components consuming JSON
+    # documents, from either Master Admin Router or Agent Admin Router).
+    gzip on;
```

Comment:
I don't think we want this. The files that are not compressed in `/assets` appear to be those that don't compress well (either due to their format already being compressed (e.g. `png`) or due to their being quite small files anyway).

I'd suggest we just want `gzip_static` and perhaps consider the additional variables mentioned at http://nginx.org/en/docs/http/ngx_http_gzip_static_module.html